ui/src/components/Client.js

```diff
@@ -19,7 +92,7 @@ class Client extends Component {
                             <ConfigurationView />
                         </TabPane>
                         <TabPane eventKey={2}>
-                            Builds
+                            <MobileClientBuildOverviewList mobileClientBuilds={mobileClientBuilds}></MobileClientBuildOverviewList>
```

Comment:
`MobileClientBuildsList` as a name?

What this list displays is a list of `buildConfigs` which can be got from `/api/buildconfigs`. Can you try using the endpoint instead of mock since it has been merged now?